_posts/2017-03-06-new-topology-system.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: A shiny, new topology system 
+---
+
+With MDAnalysis 0.16.0 on the horizon, we wanted to showcase a major development that most users will probably not notice if we've done our job well.
+In fall 2015, @richardjgowers and I set to work on redesigning the topology system from scratch.
```

Comment:
Include the image of the blackboard for some additional history...

https://pbs.twimg.com/media/CwsdYTCVQAUUQsV.jpg
https://twitter.com/orbeckst/status/795762560767180800

![image](https://cloud.githubusercontent.com/assets/237980/23393639/6f1052f2-fd41-11e6-9ebc-93f8e7f3054a.png)